e2e/command_test.go

```diff
@@ -292,6 +292,185 @@ func TestE2E_CommandTLS(t *testing.T) {
 	}
 }
 
+// TestE2E_CommandMTLSErrors tests error scenarios using CLI commands with mTLS. This
+// starts up a local Consul server and runs CTS with TLS in dev mode.
+func TestE2E_CommandMTLSErrors(t *testing.T) {
+	t.Parallel()
+
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "command_mtls_errors")
+
+	tlsc := defaultMTLSConfig()
+	cts := ctsSetupTLS(t, srv, tempDir, dbTask(), tlsc)
+
+	commands := map[string][]string{
+		"task disable": {"task", "disable"},
+		"task enable":  {"task", "disable"},
```

Comment:
Double checking if we want to test disable command 2x here

Reply:
ah copy pasta issues, I'll fix these up. Thanks for catching!